### ISSUE TITLE
-I and -C options poorly documented

### ISSUE THREAD

**Issue by sbliven** [eppic-team/eppic#72]
The -I and -C options sound like a great way to speed up the CLI for debugging:

```
  [-I <file>]  :  binary file containing the interface enumeration output of a 
                  previous run of eppic
  [-C <file>]  :  binary file containing the evolutionary scores for a particular 
                  sequence output of a previous run of eppic
```

However, shouldn't there be another set of options to output the binaries in the first place? Or even better, the first run should generate the file required by -I and subsequent runs should read it.

**Comment by sbliven** [comment]
Looks like those were removed from 3.0?

**Comment by josemduarte** [comment]
The intention of those was exactly speeding up of debugging and testing. We used them a lot in the early development before version 2 came out. However after the port to Biojava, I haven't tested them at all and I fear they won't even work. 

The way it worked (what I remember) is that with default options the serialized files would be produced, then you would read them back with these options. I think I needed to remove them completely in the port to Biojava, because of serialization issues. I'll check the code.

The whole thing need to be re-thought in any case... we should reconsider whether we want this at all and whether this is the best way of implementing it.
